### ISSUE TITLE
TypeError removing ChartCanvas Component from DOM

### ISSUE THREAD

**Issue by desy83** [rrag/react-stockcharts#421]
**Setup:**
- react-stockcharts@0.7.0-beta.22
- d3-format@1.2.1
- react@15.6.2
- Chrome Version 63.0.3239.84 (Official Build) (64-bit)
- Firefox 57.0.1 (64-bit)

**Error Stacktrace:**
```
d3-selection.js:213 Uncaught TypeError: Cannot read property 'ownerSVGElement' of null
    at point (d3-selection.js:213)
    at mouse (d3-selection.js:229)
    at EventCapture.handleMouseMove (EventCapture.js:214) or
    at eval (d3-selection.js:124)
```

**Howto reproduce:**
Insert react stock charts "ChartCanvas" into your DOM, trigger a rerender (state update, store update, rerender, ...) which removes the react stock chart component from DOM. Moving your mouse cursor will raise that TypeError.

```
array.length 
? <Chart type="hybrid" data={data} /> 
: <div className="chart-no-data-available"></div>
```

Is there anything i have to consider if i remove the react stock chart component from DOM?

**Comment by rrag** [comment]
Looks like the same issue as addressed in the PR - https://github.com/rrag/react-stockcharts/pull/420

that fix is not published to npm yet

**Comment by desy83** [comment]
Thanks rrag it is the same issue which is addressed in PL-#420. When could i expect a new release including this bugfix?
